contributing-guides/style-guide.md

```diff
@@ -104,40 +104,74 @@ Example:
 Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 ```
 
-## Token syntax
+## Placeholder syntax
 
-User-provided values should use the `{{token}}` syntax
+User-provided values should use the `{{placeholder}}` syntax
 in order to allow `tldr` clients to highlight them.
 
-Keep the following guidelines in mind when choosing tokens:
+### General rules
 
-### Naming
-- Use short but descriptive tokens,
-  such as `{{source_file}}` or `{{wallet.txt}}`.
-- Use [`snake_case`](https://wikipedia.org/wiki/snake_case) for multi-word tokens.
-- Use an actual value rather than a generic placeholder where appropriate.
-  For example, use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
+- Use short but descriptive placeholders,
+  such as `{{path/to/source.cs}}` or `{{drive}}`.
+- Use [snake_case][snake_case] for multi-word placeholders.
+- Use an actual value in placeholder such as `{{5}}` rather than
+  a placeholder with a value description inside it such as `{{seconds}}`
+  when one of the following conditions are met:
+  - there is a long (consisting of several letters) option immidiately before
+    this placeholder 
+  - there is a short (consisting of one letter) option immidiately before this
+    placeholder with a mnemonic provided via square brackets in a code
+    example description
+  - the command or it's the most nested subcommand performs one simple task such as `sleep` 
+   
+  For example, use `iostat {{2}}` rather than `iostat {{seconds}}`.
+
+> :bulb: More formal rules are described below.
+
+[snake_case]: https://wikipedia.org/wiki/snake_case
```

Comment:
This is extremely specific. It's difficult to capture a set of rules here, as we generally take it on a case-by-case basis - i.e. where it makes sense. Remember: the style guide is **not** meant to contain a comprehensive list of *all* our styling guidelines. It's meant to give a quick overview for new users.

Let's revert this.

Reply:
Where can I put comprehensive set of rules then? I can move it to a separate file. But how to name it?

Reply:
I've updated (#8366) CONTRIBUTING.md: added info about placeholders. I think it's enough to quickly start writing `tldr` pages. The comprehensive list of formal rules can be put here, in style guide.